### ISSUE TITLE
windows 10 can`t find the batch file path

### ISSUE THREAD

**Issue by AhmedaliElgabry** [iperov/DeepFaceLab#903]
## Expected behavior
The expected behavior is when double click in the batch file 2) extract images from video data_src.bat is to run and extract framed and save them in workspace/data_src

## Actual behavior
when double click in any batch file it output a message that it can`t find the batch file in the specified path.

## Steps to reproduce
i tried to run the batch file as administrator ,the result is it`s opening and closing immediately without any change happen.
and tried to install different releases from deepfacelab and the problem is the same, and tried to but the deepfacelab in different directory in my pc with no result.

appreciated any help ,thanks

![Screenshot (973)](https://user-images.githubusercontent.com/53542478/93729541-4734e780-fbc5-11ea-8713-a3956554e6c1.png)

**Comment by HotDenim** [comment]
Install to the logged in user's My Documents folder. It is known to work there atleast.

**Comment by AhmedaliElgabry** [comment]
I tried that before “install deepfacelab in Documents folder,same problem was happening

**Comment by HotDenim** [comment]
Try executing from Command Line interface, i.e. CMD console shell.

e.g:
`G:`
`CD \DeepFaceLab_NVIDIA`
`2) Extract images from` etc.......


When typing the name of the BAT file , you can use the TAB key to cycle throught and AutoComplete the filename, type a few of the first letters and then press the TAB key.

**Comment by AhmedaliElgabry** [comment]
i have tried it and that`s the output
![Screenshot (1004)](https://user-images.githubusercontent.com/53542478/94090858-ffa59a00-fe16-11ea-95c4-075eb7d3d580.png)

**Comment by jknight44** [comment]
Try switching to US English as the Windows default language and keyboard layout. I'm assuming Arabic is the default based on the date modified column.

**Comment by AhmedaliElgabry** [comment]
I have checked it ,the default language is US English

**Comment by HotDenim** [comment]
Look into the default _Environment Variable_ PATH's on your system.

Ensure they are set up to the defaults. I.e. ensure the  _\Windows_ and _\Windows\System_ folders are in the default _PATH_ Environment Variable 

Search Internet on how to do this for your particular System.

**Comment by HotDenim** [comment]
A quick fix to see if  _Environment Variables_ are the problem:

Create a new Windows Login User Acccount.  

In the New account Install DeepFaceLab.

See if it works.

**Comment by HotDenim** [comment]
Any success on the _Environment Variables_ method for the fix?

**Comment by AhmedaliElgabry** [comment]
no,i have setup another windows 10 version and it worked correctly

**Comment by ghost** [comment]
Issue solved / already answered (or it seems like user error), please close it.
